RachealN/Web-Practical
# #167956130 Add responsiveness to the Web API User Interface for good user experience and presentation

What does this PR do?
Add responsiveness to the Web API User Interface for good user experience and presentation

Description of Task to be completed?
Using media queries to add responsiveness to the web page

How should this be manually tested?
- git clone this repo:https://github.com/RachealN/Web-Practical.git and 
- navigate to index.html file
- Right-click and use inspect the property to check responsiveness

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
#167956130 

Screenshots (if appropriate)

responsive page

![respo](https://user-images.githubusercontent.com/41440727/63206710-52643c80-c0c2-11e9-9fe1-09ce276e0d8a.PNG)

media queries

![media](https://user-images.githubusercontent.com/41440727/63206719-6c9e1a80-c0c2-11e9-9c33-cc90e1bbfc2c.PNG)
